browser_tests/tests/templates.spec.ts

```diff
@@ -101,48 +107,76 @@ test.describe('Templates', () => {
     expect(await comfyPage.templates.content.isVisible()).toBe(true)
   })
 
-  test('Uses title field as fallback when the key is not found in locales', async ({
+  test('Uses proper locale files for templates', async ({ comfyPage }) => {
+    // Set locale to French before opening templates
+    await comfyPage.setSetting('Comfy.Locale', 'fr')
+
+    // Capture request for the French index file
+    let frenchIndexRequested = false
+    await comfyPage.page.route('**/templates/index.fr.json', async (route) => {
+      frenchIndexRequested = true
+      // Return the actual French index file
+      await route.continue()
+    })
```

Comment:
This might be simpler with [a waitForRequest](https://playwright.dev/docs/events#waiting-for-event)